pycparser/ply/__init__.py

```diff
@@ -1,4 +1,5 @@
 # PLY package
 # Author: David Beazley (dave@dabeaz.com)
 
+__version__ = '3.7'
```

Comment:
Not sure why this still says 3.7, but it's legit: https://github.com/dabeaz/ply/blob/master/ply/__init__.py#L4